contrib/kubernetes/README.md

```diff
@@ -0,0 +1,44 @@
+# Kubernetes  Setup for DataHub
+
+## Introduction
+This directory provides the Kubernetes setup for DataHub.  This is the first version with simple YAML files.  
+The next version will contain DataHub [Helm](https://helm.sh/) chart that can be published to [Helm Hub](https://hub.helm.sh/)
+
+## Setup
+This kubernetes deployment doesn't contain the below artifacts. The idea is to use the original helm charts for deploying each of these separately.   
+
+* Kafka and Schema Registry [Chart Link](https://hub.helm.sh/charts/incubator/kafka)
+* Elasticsearch [Chart Link](https://hub.helm.sh/charts/elastic/elasticsearch)
```

Comment:
I don't see Elastichsearch version `5.6.8` which we currently depend. For the images which are not available in Helm Hub, can we pull them from Docker Hub which is probably richer?

Reply:
1. Generally we can pass override version of any value like [imageTag](https://github.com/elastic/helm-charts/blob/master/elasticsearch/values.yaml#L46) while installing the helm chart.  Not tried yet, but something like this should work
`helm install --name elasticsearch elastic/elasticsearch --set imageTag=x.y.z`